.github/CHANGELOG.md

```diff
@@ -378,9 +382,7 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 This release contains contributions from (in alphabetical order):
 
 Marius Aglitoiu, Vishnu Ajith, Thomas Bromley, Jack Ceroni, Miruna Daian, Olivia Di Matteo,
-Tanya Garg, Christian Gogolin, Diego Guala, Anthony Hayes, Ryan Hill, Josh Izaac, Pavan Jayasinha, Ryan Levy, Nahum Sá, Maria Schuld, 
-Johannes Jakob Meyer, Brian Shi, Antal Száva, David Wierichs, Vincent Wong, Alberto Maldonado.
-
+Tanya Garg, Diego Guala, Anthony Hayes, Josh Izaac, Pavan Jayasinha, Ryan Levy, Nahum Sá, Johannes Jakob Meyer, Brian Shi, Antal Száva, David Wierichs, Vincent Wong, Alberto Maldonado, Ashish Panigrahi.
```

Comment:
Hi @paniash, it seems that new additions to the `master` branch are causing a conflict here (and as a result some contributors are missing from the PR). Could you perhaps change the list locally to the one that is there in `master` and then add your name where it'd fit in alphabetical order? :slightly_smiling_face:

Reply:
Sorry about that! It should be fixed now. :-)

Reply:
Looks like there is some additional merge conflict that arises. Sorry about this. I'll fix it as soon as I can.

Reply:
That's okay, no worries! Locally, once thing that can be done is to pull `master`, switch to the dedicated branch and then go for `git merge master`. This will merge the changes from master and show if there are conflicts for each file.

Then going through each file with conflicts and resolving them helps. `git` will put `HEAD` and `master` markers (the branch name with which there's a conflict) in the file with visual lines too. Resolving the conflict goes as comparing the version of the source file from the pull request with the version of the source file on `master`. Some conflicts can be non-trivial, let us know if something like this comes up.

Reply:
@antalszava Sorry about the delay, was caught up with some personal stuff. I've fixed the merge conflict. Let me know if it's alright. :-)